sources/packages/web/src/components/aest/students/bypassedRestrictions/HistoryBypassedRestrictions.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+  <v-card>
+    <v-container>
+      <body-header title="History of Bypassed Restrictions" class="m-1">
+        <template #actions>
+          <check-permission-role :role="Role.StudentAddRestriction">
```

Comment:
`Role.StudentAddRestriction` role is not part of the ticket, please refrain from using that. 

Also the action button `Bypass A Restriction` can be left as dummy as possible(without being wrapped by check-permission-role) or even not existing.

Reply:
Removed.